volatility3/cli/text_renderer.py

```diff
@@ -165,7 +165,10 @@ def render(self, grid: interfaces.renderers.TreeGrid) -> None:
         """
         # TODO: Docstrings
         # TODO: Improve text output
-        outfd = sys.stdout
+        if self.output_filename:
```

Comment:
Since this is a repetitive change of code, it suggests I didn't do a great job when designing these classes, it probably should have been an input parameter you could provide somewhere, but not to worry.  5:)

Reply:
This code should probably be drawn out into a class method, so it can be reused by all the renderers, which will also mean there's only one place to update given this needs to check whether the file already exists and then decide to write to it, rename the file or abort (and giving the user a message saying which it did).